test/app/src/getOktaSignIn_CDN.ts

```diff
@@ -0,0 +1,11 @@
+declare global {
```

Comment:
Can we change the file naming convention like `OktaSignInCDN.ts` or any other you'd think of without mixing `_` and verbs.

Reply:
change to `getOktaSingInWIdgetFromCDN`